dots_server/db_models/template.sql

```diff
@@ -5,7 +5,7 @@ DROP TABLE IF EXISTS `blocker`;
 
 CREATE TABLE `blocker` (
   `id` bigint(20) NOT NULL AUTO_INCREMENT,
-  `type` VARCHAR(20) NOT NULL,
+  `blocker_type` VARCHAR(255) NOT NULL,
```

Comment:
This matches to the type len used elsewhere.

Reply:
As you said, column name `blocker_type` is correct. and VARCHAR(255) is better length.
And `blocker_type` in test_dump.sql should be VARCHAR(255) too.